src/fsharp/ErrorResolutionHints.fs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+module internal Microsoft.FSharp.Compiler.ErrorResolutionHints
+
+let CreatePredictionText textF unknownIdent predictions =
```

Comment:
Please add a ``///`` comment to all new top-level functions, modules and types, thanks